client/src/styles/color-palette.scss

```diff
@@ -0,0 +1,20 @@
+// GENERAL
+$color-white: #ffffff;
+$color-black: #000000;
+
+// BACKGROUNDS
+$primary-background: #fafafa;
+$secondary-background: #f4f4f4;
+
+// BORDERS
+$primary-border-color: #dbdbdb;
+$secondary-border-color: #a8a8a8;
+
+// TEXT
+$primary-text-color: #262626;
+$secondary-text-color: #8e8e8e;
+
+// Call to action
+$primary-color: #0095f6;
+$secondary-color: #0095f64d;
+$active-color: #41b0fa;
```

Comment:
I think a better name for this color is $primary-light maybe. 
It is more of a shade of the primary color then a color of its own